sources/netdog/src/dns.rs

```diff
@@ -277,82 +200,6 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "wicked")]
-    fn dns_from_lease_file() {
-        let lease_path = test_data().join("leaseinfo.eth0.dhcp.ipv4");
```

Comment:
There is a `test_data` directory that should have all the wicked files removed as well.